hardware/raspberrypi/bootmodes/bootflow.md

```diff
@@ -44,10 +38,10 @@ NOTES:
 * If there is no SD card inserted, the SD boot mode takes five seconds to fail. To reduce this and fall back to USB more quickly, you can either insert an SD card with nothing on it or use the GPIO bootmode OTP setting described above to only enable USB.
 * The default pull for the GPIOs is defined on page 102 of the [ARM Peripherals datasheet](../bcm2835/BCM2835-ARM-Peripherals.pdf). If the value at boot time does not equal the default pull, then that boot mode is enabled.
 * USB enumeration is a means of enabling power to the downstream devices on a hub, then waiting for the device to pull the D+ and D- lines to indicate if it is either USB 1 or USB 2. This can take time: on some devices it can take up to three seconds for a hard disk drive to spin up and start the enumeration process. Because this is the only way of detecting that the hardware is attached, we have to wait for a minimum amount of time (two seconds). If the device fails to respond after this maximum timeout, it is possible to increase the timeout to five seconds using `program_usb_boot_timeout=1` in `config.txt`.
-* MSD takes precedence over Ethernet boot.
+* MSD boot takes precedence over Ethernet boot.
 * It is no longer necessary for the first partition to be the FAT partition, as the MSD boot will continue to search for a FAT partition beyond the first one.
 * The boot ROM also now supports GUID partitioning and has been tested with hard drives partitioned using Mac, Windows, and Linux.
-* The LAN951x is detected using the Vendor ID 0x0424 and Product ID 0xec00, this is different to the standalone LAN9500 device which has a product ID of 0x9500 or 0x9e00.  To use the standalone LAN9500 an I2C EEPROM would need to be added to change these ID's to match the LAN9514
+* The LAN951x is detected using the Vendor ID 0x0424 and Product ID 0xec00: this is different to the standalone LAN9500 device which has a product ID of 0x9500 or 0x9e00.  To use the standalone LAN9500 an I2C EEPROM would need to be added to change these ID's to match the LAN9514. (Similarly for the LAN7500).
```

Comment:
@lurch Instead of "to change these ID's to match the LAN9514", would "[...] the LAN951x" be more accurate?

Reply:
I'm not sure - some Pi's use the LAN9512, some use the LAN9514 and some use the LAN7515, and I'm not sure what VendorID and ProductID these all appear as.

ping @ghollingworth for clarification

Reply:
PI 1 used 9512, which appears as 0xec00
Pi 2 and 3 used 9514 which appears as 0xec00
Pi 3B+ use 7515 which appears as 0x7800

A standalone 9500 would appear as 0x9500 or 0x9e00 and wouldn't work (so requires EEPROM)

A standalone 7800 would appear as 0x7800 and therefore would work fine without an EEPROM

Reply:
@helenlynn Based on Gordon's answer, I guess you can use either "the LAN9514" or "the LAN951x" - whichever you think sounds better. And it seems the "(Similarly for the LAN7500)." addition isn't necessary.